src/gridcoin/scraper/scraper_net.cpp

```diff
@@ -139,9 +139,7 @@ int CSplitBlob::addPartData(CDataStream&& vData)
     {
         /* missing data; use the supplied data */
         /* prevent calling the Complete callback FIXME: make this look better */
-        cntPartsRcvd--;
         CSplitBlob::RecvPart(nullptr, vData);
-        cntPartsRcvd++;
```

Comment:
I think your removal is correct. I cannot for the life of me determine why we did that.